File: src/main/paradox/includes/code/FormingACluster.scala
package sdocs.includes

import akka.actor.ActorSystem

object StartClusterBootStrap {

  val actorSystem = ActorSystem()

  // #start
  import akka.management.scaladsl.AkkaManagement
  import akka.management.cluster.bootstrap.ClusterBootstrap

  AkkaManagement(actorSystem).start()
  ClusterBootstrap(actorSystem).start()
  // #start


}

File: src/main/paradox/includes/code/jdocs/includes/FormingACluster.java
package jdocs.includes;

import akka.actor.ActorSystem;
// #start
import akka.management.AkkaManagement;
import akka.management.cluster.bootstrap.ClusterBootstrap;
// #start

public interface FormingACluster {
    class StartClusterBootstrap {
        {
            ActorSystem actorSystem = ActorSystem.create();

            // #start

            AkkaManagement.get(actorSystem).start();
            ClusterBootstrap.get(actorSystem).start();
            // #start
        }
    }
}

